src/extractor/guidance/turn_analysis.cpp

```diff
@@ -184,14 +169,12 @@ bool TurnAnalysis::isRotary(const NodeID nid) const
     // with both vertices right at the other side (so half their distance in meters).
     // Otherwise, we construct a circle through the first tree vertices.
     auto node_itr = roundabout_nodes.begin();
-    const double radius = roundabout_nodes.size() >= 3
-                              ? util::coordinate_calculation::circleRadius(
-                                    getCoordinate(*node_itr++),
-                                    getCoordinate(*node_itr++),
-                                    getCoordinate(*node_itr))
-                              : 0.5 * util::coordinate_calculation::haversineDistance(
-                                          getCoordinate(*node_itr++),
-                                          getCoordinate(*node_itr));
+    const double radius =
+        roundabout_nodes.size() >= 3
+            ? util::coordinate_calculation::circleRadius(
+                  getCoordinate(*node_itr++), getCoordinate(*node_itr++), getCoordinate(*node_itr++))
```

Comment:
@MoKob just found this sequencing issue ^ capturing some resources from chat for the record

- http://herbsutter.com/gotw/_102/
- http://herbsutter.com/2014/12/01/a-quick-poll-about-order-of-evaluation/

@TheMarex @danpat maybe you're interested.